index.html

```diff
@@ -6,4 +6,7 @@
 pg: 0
 title: About Us
 ---
-CTFs are security events in which you normally have to solve challenges in order to obtain flags. Most of these CTFs require you to have a team which prompted us to create Crackchester as an academic CTF team.
+<h1>What is CTF?</h1>
```

Comment:
Not sure if it is a good idea to have this as the only title.

After all this is about us as a team rather than CTFs in general.
Perhaps rewriting the paragraph and changing the heading to "What is Crackchester?" would be better.

Reply:
I'll leave the content up to you, the title was just based on what you'd written so far.